ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py

```diff
@@ -613,3 +613,541 @@ def test_jax_special_rrshift(
             ret_from_gt=ret_gt,
             ground_truth_backend="jax",
         )
+
+
+# __add__
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric", full=True),
+        shared_dtype=True,
+        num_arrays=2,
+    )
+)
+def test_jax_special_add(
+    dtype_x,
+    fw,
+):
+    input_dtype, x = dtype_x
+    ret = DeviceArray(x[0]) + DeviceArray(x[1])
+    ret_gt = jnp.array(x[0], dtype=input_dtype[0]) + jnp.array(
+        x[1], dtype=input_dtype[1]
+    )
+    ret = helpers.flatten_and_to_np(ret=ret)
+    ret_gt = helpers.flatten_and_to_np(ret=ret_gt)
+    for (u, v) in zip(ret, ret_gt):
+        helpers.value_test(
+            ret=ret,
+            ret_from_gt=ret_gt,
+            ground_truth_backend="jax",
+        )
+
+
+# __radd__
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric", full=True),
+        shared_dtype=True,
+        num_arrays=2,
+    )
+)
+def test_jax_special_radd(
+    dtype_x,
+    fw,
+):
+    input_dtype, x = dtype_x
+    data = DeviceArray(x[0])
+    other = DeviceArray(x[1])
+    ret = data.__radd__(other)
+    ret_gt = jnp.array(x[0], dtype=input_dtype[0]) + jnp.array(
```

Comment:
is it possible to do:
`ret_gt = jnp.array(x[1], dtype=input_dtype[1]).__radd__(jnp.array(x[0], dtype=input_dtype[0]))`

just like the `ret` computation


(this applies to all `__r<>__` methods testing)